src/main/java/de/metas/ui/web/window/model/DocumentStandardAction.java

```diff
@@ -43,12 +42,13 @@
 	Email("email"), //
 	Letter("letter"), //
 	Print("print"), //
-	Delete("delete") //
+	Delete("delete"), //
+	Notes("notes")//
```

Comment:
"don't pun", i.e. call all of it "comment" or "note", unless there is a really good reason..

Reply:
Probably dumb question, but why use singular?

Reply:
Missunderstanding on my part: I forgot to rename to `Comments` instead of `Notes`